cloud-minecraft/cloud-brigadier/src/main/java/cloud/commandframework/brigadier/CloudBrigadierManager.java

```diff
@@ -192,6 +192,70 @@ private void registerInternalMappings() {
         }, false, argument -> StringArgumentType.greedyString());
     }
 
+    /**
+     * Set whether to use Brigadier's native suggestions for number argument types
```

Comment:
Might be worth clarifying that *our* suggestions will be used otherwise (i.e. number suggestions)